framework/tests/viewpointtest/SuperConstructorCalls.java

```diff
@@ -0,0 +1,51 @@
+// Test case for EISOP issue #782:
+// https://github.com/eisop/checker-framework/issues/782
+import viewpointtest.quals.*;
+
+public class SuperConstructorCalls {
+
+    public SuperConstructorCalls() {}
+
+    @SuppressWarnings({"inconsistent.constructor.type", "super.invocation.invalid"})
+    public @ReceiverDependentQual SuperConstructorCalls(@ReceiverDependentQual Object str) {}
+
+    @SuppressWarnings({"inconsistent.constructor.type", "super.invocation.invalid"})
+    public @A SuperConstructorCalls(@A float number) {}
+
+    class Inner extends SuperConstructorCalls {
+        public Inner() {}
+
+        public Inner(Object obj) {
+            super();
+        }
+
+        public Inner(int number, @Top Object obj) {
+            super(obj);
+        }
+
+        // :: warning: (inconsistent.constructor.type)
```

Comment:
When do you expect a warning and when do you use a `@SuppressWarnings`?
The warning here is for the same reason as above, where you used `@SuppressWarnings`, so for consistency it would be good to use the same style.